test/smoke/harness/backend.go

```diff
@@ -45,7 +52,7 @@ func (h *Harness) watchBackendLogs(r io.Reader) {
 				// ignore printed SQL errors
 				continue
 			}
-			h.t.Errorf("Backend: %s(%s) %s", strings.ToUpper(entry.Level), entry.Source, entry.Error)
+			h.t.Errorf("Backend: %s(%s) %s: %s\n%s", strings.ToUpper(entry.Level), entry.Source, entry.Error, entry.Message, string(raw))
```

Comment:
would a newline cause visualization issues?  I know with TAP the error message should be on a single line, or each line gets written to a new log entry, leading to confusion.

Reply:
This is only used by the harness when reporting in tests (`testing.T.Errorf`); never comes from the application itself

If a test fails, you'll see the short error message inline, followed by the JSON blob on it's own that may have more details on what went wrong